code/index.html

```diff
@@ -5,32 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Johannas Project Cluedo</title>
   </head>
   <body>
+    <h1>Welcome to the game Cluedo</h1>
+    <h3 class="text">Play by clicking on the cards.</h3>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p><strong>KILLER</strong></p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
         <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
       </article>
       <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
+        <p><strong>WEAPON</strong></p>
+        <p id="weaponName" style = "text-transform:capitalize;"></p>
         <p id="weaponWeight"></p>
       </article>
       <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
+        <p><strong>ROOM</strong></p>
         <p id="roomName"></p>
       </article>
     </section>
 
-    <h1 id="mystery"></h1>
+    <button onclick="revealMystery()" id="button">REVEAL THE CRIME</button>
```

Comment:
Great that you try out two ways of calling a function upon an event - both eventlistener in javasript and the onclick here in html! At least that is my perspective as we are learning :-)